js/src/components/app-text/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
+
+/**
+ * Renders a paragraph with some predefined styles based on the variant
+ *
+ *
+ * ```jsx
+ * <AppText variant="title-small">
+ * 		My text
+ * </AppText>
+ * ```
+ *
+ * @param {Object} props Component props
+ * @param {string} [props.variant] The variant to use for the text
+ * @param {string} [props.className] Custom classname for this component
+ * @param {string} [props.as] as Custom HTML tag for the component. For example: p, h1, h2, h3, span... (by default <p>)
```

Comment:
:nail_care: We can denote the default value as more conventional. It's the tag name, I'm not sure how it would behave if someone would provide `<p>` in HTML/JSX, plus my … a11y OCD ;)
```suggestion
 * @param {string} [props.as='p'] as Custom HTML tag name for the component. For example: p, h1, h2, h3, span…
```